docker/elasticsearch-setup/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Defining environment
 ARG APP_ENV=prod
 
-FROM alpine:3 as base
+FROM --platform=$BUILDPLATFORM alpine:3
```

Comment:
@pedro93 I think this is a typo here. You missed `as base`

Reply:
Good catch! Thanks Aseem!